### ISSUE TITLE
@typescript-eslint/no-unsafe-call with svelte-i18n and @typescript-eslint/recommended-requiring-type-checking

### ISSUE THREAD

**Issue by binarious** [ota-meshi/eslint-plugin-svelte#195]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

8.20.0

### What version of `eslint-plugin-svelte` are you using?

2.2.0

### What did you do?

<details>
<summary>Configuration</summary>

```js
module.exports = {
  parser: '@typescript-eslint/parser',
  extends: [
    'plugin:svelte/recommended',
    'eslint:recommended',
    'plugin:@typescript-eslint/recommended-requiring-type-checking'
  ],
  parserOptions: {
    project: ['./tsconfig.json'],
    extraFileExtensions: ['.svelte']
  },
  overrides: [
    {
      files: ['*.svelte'],
      parser: 'svelte-eslint-parser',
      parserOptions: {
        parser: "@typescript-eslint/parser",
      },
    },
  ],
  ignorePatterns: ['node_modules', 'dist']
}
```
</details>

```svelte
<script lang="ts">
  import { _ } from 'svelte-i18n';
</script>

<main>
  <input name={$_('test')}>
</main>
```

### What did you expect to happen?

No lint issues to occur.

### What actually happened?

> Unsafe call of an `any` typed value. [@typescript-eslint/no-unsafe-call](https://typescript-eslint.io/rules/no-unsafe-call)

for `$_('test')`. This doesn't happen with https://github.com/sveltejs/eslint-plugin-svelte3.

### Link to Minimal Reproducible Example

https://github.com/binarious/eslint-plugin-svelte-repro

**Comment by ota-meshi** [comment]
Thank you for posting this issue.

It seems that typescript will judge if it is an `any` type because it cannot get the type of `$_` correctly. I'll try to find a way around it, but in the meantime I recommend using `settings.svelte.ignoreWarnings` or turning off the rule for `*.svelte` files.

https://github.com/ota-meshi/eslint-plugin-svelte#settingssvelte
https://eslint.org/docs/latest/user-guide/configuring/configuration-files#how-do-overrides-work

**Comment by binarious** [comment]
> I'll try to find a way around it

Thank you!

> in the meantime I recommend using settings.svelte.ignoreWarnings or turning off the rule

Actually this results in multiple rules being violated depending on the use-case.

I just realized, that the issue is with all usages of svelte's `$` - e. g. getting values from a store via `$myStore.property` results in `any`.
